tslint.json

```diff
@@ -72,16 +79,26 @@
     // Bans jasmine helper functions that will prevent the CI from properly running tests.
     "ban": [
       true,
-      ["fit"],
-      ["fdescribe"],
-      ["xit"],
-      ["xdescribe"]
+      [
+        "fit"
+      ],
+      [
+        "fdescribe"
+      ],
+      [
+        "xit"
+      ],
+      [
+        "xdescribe"
+      ]
```

Comment:
I think the original formatting was fine. Just add `no-unescaped-html-tag` without any of the other changes.

Reply:
Oh, that was and accidental `shift+alt+f` in Visual Studio Code.